node-repository/src/test/java/com/yahoo/vespa/hosted/provision/maintenance/HostCapacityMaintainerTest.java

```diff
@@ -316,6 +320,44 @@ public void respects_exclusive_allocation() {
         tester.assertNodesUnchanged();
     }
 
+    @Test
+    public void works_as_before_without_make_exclusive() {
```

Comment:
What about a test with `make-exclusive`?

Reply:
The above test has that.